graphql/cxs-impl/src/main/java/org/apache/unomi/graphql/fetchers/CustomerPropertyDataFetcher.java

```diff
@@ -19,21 +19,25 @@
 import graphql.schema.DataFetchingEnvironment;
 import org.apache.unomi.graphql.schema.PropertyNameTranslator;
 import org.apache.unomi.graphql.types.output.CDPEvent;
+import org.apache.unomi.graphql.types.output.CDPPersona;
 import org.apache.unomi.graphql.types.output.CDPProfile;
 
-public class CustomerPropertyDataFetcher extends BaseDataFetcher<Object> {
+public class CustomerPropertyDataFetcher extends DynamicFieldDataFetcher<Object> {
 
     private final String propertyName;
```

Comment:
please remove that property, and use `super(PropertyNameTranslator.translateFromGraphQLToUnomi(propertyName))`

Reply:
No! Default constructor of DynamicFieldDataFetcher should always accept field name as string.